resources/js/components/Kiosk.vue

```diff
@@ -533,7 +543,7 @@ export default {
             } else {
                 this.$swal.fire(
                     'Error',
-                    'Unable to process data. Check your internet connection or try refreshing the page.',
+                    'An unexpected error occurred. If this continues, post in Slack or contact developers@robojackets.org.',
```

Comment:
Should we specify a channel to post in Slack for people who are unfamiliar?

Reply:
I debated that because it'll mean different things to different people.
Thoughts on this?
>If this continues, post in SCC/RJ Slack #people-counter or contact developers@robojackets.org